lib/rules/eslint-plugin-disallow-single-sign-variable-in-arrow-expressions.js

```diff
@@ -9,34 +9,33 @@
 //------------------------------------------------------------------------------
 
 module.exports = {
-    meta: {
-        docs: {
-            description: "Disallow single sign variable in arrow expressions",
-            category: "Fill me in",
-            recommended: false
-        },
-        fixable: null,
-        schema: []
+  meta: {
+    docs: {
+      description: 'Disallow single sign variable in arrow expressions',
+      category: 'Stylistic Issues',
+      recommended: false
     },
+    fixable: null,
+    schema: []
+  },
 
-    create: function(context) {
-        return {
-            ArrowFunctionExpression(node) {
-                if (context.getFilename().includes('.test.')) {
-                    return undefined;
-                }
-                node.params.forEach(param => {
-                    if (param.type === 'Identifier' && param.name.length === 1) {
-                        context.report({
-                            node: node,
-                            message:
-                                'Unexpected non-descriptive parameter received in arrow function expression - "{{ e }}"',
-                            data: { e: param.name.trim() }
-                        });
-                    }
-                })
-            }
-
-        };
-    }
+  create: function(context) {
+    return {
+      ArrowFunctionExpression(node) {
+        if (context.getFilename().includes('.test.')) {
```

Comment:
I think this check could be even before checking for ArrowFunctionExpression but it shouldn't matter much (maybe file parsing speed?)